src/libraries/Microsoft.Extensions.Options.ConfigurationExtensions/ref/Microsoft.Extensions.Options.ConfigurationExtensions.cs

```diff
@@ -10,6 +10,7 @@ public static partial class OptionsBuilderConfigurationExtensions
     {
         public static Microsoft.Extensions.Options.OptionsBuilder<TOptions> Bind<TOptions>(this Microsoft.Extensions.Options.OptionsBuilder<TOptions> optionsBuilder, Microsoft.Extensions.Configuration.IConfiguration config) where TOptions : class { throw null; }
         public static Microsoft.Extensions.Options.OptionsBuilder<TOptions> Bind<TOptions>(this Microsoft.Extensions.Options.OptionsBuilder<TOptions> optionsBuilder, Microsoft.Extensions.Configuration.IConfiguration config, System.Action<Microsoft.Extensions.Configuration.BinderOptions> configureBinder) where TOptions : class { throw null; }
+        public static Microsoft.Extensions.Options.OptionsBuilder<TOptions> BindConfiguration<TOptions>(this Microsoft.Extensions.Options.OptionsBuilder<TOptions> optionsBuilder, string configSectionPath, System.Action<Microsoft.Extensions.Configuration.BinderOptions> configureBinder = null) where TOptions : class { throw null; }
```

Comment:
nit: when making updates to ref files, it's recommended to use the GenerateReferenceAssemblySource target like below (skip -c Release if not building in Release.), after you're done building the src project and before building ref: 

> dotnet build src\libraries\Microsoft.Extensions.Options.ConfigurationExtensions\src\ /t:GenerateReferenceAssemblySource -c Release

That is better than needing to do manual edits. This result should bring this new line up to line 11 instead. 

The general workflow is, you build in src csproj, then run GenerateReferenceAssemblySource target, then build ref project (Link to [guideline](https://github.com/dotnet/runtime/blob/2a4284b9f3eb2ad95fdf324f0249d372abff96df/docs/coding-guidelines/updating-ref-source.md))

Reply:
sorry about that, I fixed that in b9628cc4e520c0a49d76240824d96427703aca6e